repo/packages/K/kubernetes/1/config.json

```diff
@@ -81,6 +81,15 @@
           "minimum": 0,
           "type": "integer"
         },
+        "resource-roles": {
```

Comment:
I was thinking apiserver-resource-roles, but I'm not sure that's much clearer, given that the other apiserver configs are flags sent to apiserver.